web/modules/custom/joinup_licence/src/Form/CompatibilityDocumentForm.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_licence\Form;
+
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form controller for the compatibility document entity edit forms.
+ */
+class CompatibilityDocumentForm extends ContentEntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    $entity = $this->getEntity();
+    $entity->save();
```

Comment:
Instead of doing this, I'd prefer to call the parent method.

Reply:
This code is originating from the [drupal-code-generator](https://github.com/Chi-teck/drupal-code-generator/blob/cf346f6f67cff0253f9a2c8c36f78fb4103215b2/tests/dcg/Generator/_content_entity/src/Form/ExampleForm.php#L19) but I checked and it is not a problem upstream.